ethicml/data/tabular_data/admissions.py

```diff
@@ -37,19 +37,31 @@
 }
 ```
 """
-
+from enum import Enum
+from typing import Union
 from typing_extensions import Literal
 
 from ..dataset import Dataset
 from ..util import flatten_dict
 
 __all__ = ["admissions"]
 
-AdmissionsSplits = Literal["Gender"]
 
+class AdmissionsSplits(Enum):
+    GENDER = "Gender"
```

Comment:
All caps for the values?

Reply:
Apparently all caps for the name is the done thing according to pylint. I'm not sure I like it